### PR TITLE
Update Nomis alert mapping July 2020 + small refactor

### PR DESCRIPTION
**Pull request by alexdesi**
### Jira link

I am working on https://dsdmoj.atlassian.net/browse/P4-1933
This PR tries to make `NomisAlerts::Importer` a bit more readable

### What?
NomisAlerts::Importer

### Why?
to make this cleaner in order to work on the P4-1933

### CODE REVIEW

File: app/services/nomis_alerts/importer.rb [hunk 1]
```diff
@@ -128,30 +170,42 @@ def call
...
-      Rails.logger.info "Missing alert type #{alert[:parent_code]}" if alert_type.nil?
-      return if alert_type.nil?
 
-      record = NomisAlert.find_or_initialize_by(code: alert[:code], type_code: alert[:parent_code])
-      record.update!(
-        description: alert[:description],
-        type_description: alert_type[:description],
-        assessment_question: assessment_question_mapping(alert[:code]),
-      )
+      if alert_type_for(alert).nil?
```
Comment (willfish) [inline app/services/nomis_alerts/importer.rb:180]:
```suggestion
      if alert_type.nil?
```

File: app/services/nomis_alerts/importer.rb [hunk 2]
```diff
@@ -91,7 +101,7 @@ class Importer
       'XB' => :violent,
       'XC' => :escape,
       'XCU' => :escape,
-      'XSDEPORT' => :escape,
+      'XSDEPORT' => :not_to_be_released,
```
Comment (martyn-w) [inline app/services/nomis_alerts/importer.rb:104]: just flagging this - is it intentional that `XSDEPORT` be remapped from `escape` to `not_to_be_released` ?

Comment (alexdesi) [reply app/services/nomis_alerts/importer.rb:104]: yes @martyn-w, Jade asked for this chage, it is the spreadsheet
Screen Shot 2020-07-20 at 14 31 57

Comment (martyn-w) [reply app/services/nomis_alerts/importer.rb:104]: 👍 jolly good

**Pull request merged**